rope/contrib/codeassist.py

```diff
@@ -347,15 +350,26 @@ def default_templates():
     return {}
 
 
+def _startswith(s1, s2):
```

Comment:
That's ``string1``, ``string2``. Yes, I can rename it to that, but it seems silly.